package.json

```diff
@@ -33,7 +33,7 @@
     "dev:windows:dashboard": "set HOST=127.0.0.1&& set PORT=3000&& set DASHBOARD=1&& webpack-dashboard -- webpack-dev-server --config ./webpack/dev-server.js",
     "dev:windows": "set HOST=127.0.0.1&& set PORT=3000&& node ./webpack/dev-server.js",
     "lint": "eslint --ignore-path .gitignore -c .eslintrc \"./**/*.js\"",
-    "test": "mocha test/unit/ --recursive --compilers js:babel-register --reporter spec",
```

Comment:
You've outcommented unit tests by this change. Please reinclude them or introduce different tests levels (e.g. test:components, test:unit, test:...).